### ISSUE TITLE
Error in gdefs.ml while generating gadgets for SPEC binary

### ISSUE THREAD

**Issue by sriskanda88** [pakt/ropc#4]
Hi,

The tool works excellently when I use it on small target binaries but when I use it on larger binaries (SPEC) I get the same error at the same place every single time. The last few lines of the output are attached. I tried debugging it but to no avail :( 

![fatal error](https://f.cloud.github.com/assets/6365678/2107173/ba801808-8fc1-11e3-952a-eae31d98ec3b.PNG)

Any help is greatly appreciated !

Thanks,
Skanda

**Comment by pakt** [comment]
Seems like VEX library is unable to model RCR instruction correctly. Try binaries without RCR instructions.

**Comment by sriskanda88** [comment]
Yup, I tried some test binaries without RCR and it worked fine. Is there any fix or workaround? Some way to ignore all RCR and RCL instructions maybe?

**Comment by pakt** [comment]
One way to "ignore" unknowns is to replace them with constants (like 0) before passing AST statements to the symbolic evaluator. This could introduce bugs, because such behavior doesn't model code semantics correctly.
